CHANGELOG.md

```diff
@@ -35,12 +35,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## Unreleased
-
 ### Improvements
 
-- [#1234](https://github.com/babylonlabs-io/babylon/pull/1234) doc: revise zone concierge doc (round 2)
```

Comment:
is OK to remove this chlog entry?

Reply:
fixed, it was merge conflict gone bad, thanks for catching that